app/forms/portal/client_login_form.rb

```diff
@@ -14,22 +14,20 @@ def client
     private
 
     def matches_client
-      if last_four.present?
+      if number.present?
```

Comment:
Does this mean that product doesnt want to require both last 4 ssn AND confirmation number? I thought it was a both thing, not an either or thing!

Reply:
okay i checked with them and i guess it is either or. This means we need to update the macro text i was provided that says you need both 😂

Reply:
Great :) I assume you'll take care of that. :)